.github/workflows/ci-master.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
     - master
+    - feat-fake-header
```

Comment:
This PR merges `feat-fake-header` into `master`, so the branche `feat-fake-header` should become obsolete.
Correct? If so, why adding the branche `feat-fake-header` to the ci of the master branche?
(I assume this is an artifact from testing/working the branch)

Reply:
Hi MichaelPFrey, you are right, leftover from testing. Thanks for careful watching!